andela/wg-bigas-opus
# Ft export workouts 15711085

#### What does this PR do?
It implements the importing and exporting of workouts
#### Description of Task to be completed?
a user should be able to import and export workout and exerscises
#### How should this be manually tested?
after cloning and setting up the repo locally, change the branch to `ft-export-workouts-15711085` navigate to workout overview and add workout then add and training day and exercises. click on the options button you will see a export workout 
<img width="1276" alt="screen shot 2018-05-11 at 11 21 57" src="https://user-images.githubusercontent.com/19773670/39916174-71e11c80-5512-11e8-8763-4b45bff7824a.png">

it will then download a csv file to your download folder

<img width="639" alt="screen shot 2018-05-11 at 07 39 08" src="https://user-images.githubusercontent.com/19773670/39916233-9455e9ee-5512-11e8-9131-8668b24d06c8.png">

to import a csv navigate to the workout overview and adjacent to the add  button there is an import button. 
<img width="1273" alt="screen shot 2018-05-11 at 11 29 45" src="https://user-images.githubusercontent.com/19773670/39916452-2be99814-5513-11e8-91c6-0237444086d0.png">

<img width="490" alt="screen shot 2018-05-11 at 07 40 52" src="https://user-images.githubusercontent.com/19773670/39916485-43d1d414-5513-11e8-9fe0-4b89be526bde.png">

<img width="1280" alt="screen shot 2018-05-11 at 07 41 14" src="https://user-images.githubusercontent.com/19773670/39916498-4d4dc08e-5513-11e8-9886-0e893632dee2.png">
